std/range/package.d

```diff
@@ -2887,13 +2905,37 @@ private:
     UT _value;
 
 public:
-    @property inout(T) front() inout { return _value; }
-    @property inout(T) back() inout { return _value; }
+    ///
+    @property inout(T) front() inout
+    {
+        return _value;
+    }
+
+    ///
```

Comment:
> Let's wait for feedback from other people here - I am more on the side of supplying unnecessary information bloats and is less helpful.

Does anyone have an opinion on whether range primitives need a comment or just documenting it with `///` is better?